samples/bluetooth/broadcast_audio_source/boards/nrf52_bsim.conf

```diff
@@ -0,0 +1,3 @@
+# When Host is fixed, CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE can inherit the CONFIG_BT_ISO_TX_MTU value.
+CONFIG_BT_ISO_TX_MTU=40
```

Comment:
After the addition of https://github.com/zephyrproject-rtos/zephyr/pull/64635 we actually need this value to be `60` to support BAP 24_2_1

Reply:
would it be sensible to have this set in the prj.conf instead?

Reply:
Yeah, it will actually affect all platforms

Reply:
done